src/semantic_csv/impl/core.cljc

```diff
@@ -74,13 +75,14 @@
 (defn- subset-map [m ks]
   (into {} (map (fn [k] [k (get m k)]) ks)))
 
-(defmacro clone
```

Comment:
Hmm... Are we sure this won't work in cljs? Might cause problems with bootstrapped compilation I suppose, since cljs doesn't have resolve. In any case, have to make sure we manually def these for cljs (too bad we can't as easily copy over the docs though).